cmd/acr/purge.go

```diff
@@ -48,15 +50,16 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
 	acr purge -r example --filter "hello-world:.*" --ago 1d --concurrency 4
 	`
-	maxPoolSize              = 32 // The max number of parallel delete requests recommended by ACR server
-	manifestListContentType  = "application/vnd.docker.distribution.manifest.list.v2+json"
-	manifestIndexContentType = "application/vnd.oci.image.index.v1+json"
-	linkHeader               = "Link"
+	maxPoolSize                 = 32 // The max number of parallel delete requests recommended by ACR server
+	mediaTypeDockerManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+	mediaTypeArtifactManifest   = "application/vnd.oci.artifact.manifest.v1+json"
+	headerLink                  = "Link"
 )
 
 var (
 	defaultPoolSize        = runtime.GOMAXPROCS(0)
 	concurrencyDescription = fmt.Sprintf("Number of concurrent purge tasks. Range: [1 - %d]", maxPoolSize)
+	manifestMediaTypes     = set.New[string](mediaTypeDockerManifestList, mediaTypeArtifactManifest, v1.MediaTypeImageManifest, v1.MediaTypeImageIndex)
```

Comment:
I realized that it's not quite appropriate to call this `manifestMediaTypes`, as docker manifest is a manifest but it's not one of `manifestMediaTypes` which is a bit confusing🤔. But I don't have a good name in my mind yet... Or maybe we can add a few docs here.

Reply:
discussed offline, change variable name to mediaTypesWithComplexStructure. Open for suggestions.

Reply:
How about `aggregateManifestMediaTypes`?